gulp/config.js

```diff
@@ -62,12 +62,6 @@ module.exports = {
     dest: loc.dist + '/js/',
     name: 'main.js'
   },
-  browserify: {
-    paths: {
-      scripts: 'retirement_api/static/retirement/js/claiming-social-security.js',
```

Comment:
Note that this script does not appear to be referenced anywhere. Is it used??? @mistergone

Reply:
I think that is a reference to a script from an older version of this repo. I think it can be ignored/removed safely.